### ISSUE TITLE
License inconsistency

### ISSUE THREAD

**Issue by jdemeyer** [videlec/pplpy#28]
The source files say
```
#  Distributed under the terms of the GNU General Public License (GPL)
#  as published by the Free Software Foundation; either version 2 of
#  the License, or (at youroption) any later version.
#                  http://www.gnu.org/licenses/
```
but the `LICENSE.txt` is GPL version 3 and also `setup.py` says `GPL v3`.

I believe that Sage is supposed to be compatible with "GPL version 3 or later". So if possible, please use a license compatible with that.

**Comment by vinklein** [comment]
@videlec Is it okay to set version 3 as the minimal in the sources files ? I don't know if we have the rights to change this.

**Comment by jdemeyer** [comment]
> I don't know if we have the rights to change this.

Of course you do. The original sources say "GPL version 2 or later". So you can restrict it to "GPL version 3 or later".

Note: because of the GPL, you could not change it to say, a BSD license.

**Comment by vinklein** [comment]
Oh okay that make sense.

**Comment by vinklein** [comment]
Fix proposed in PR #29.
